runtime/doc/map.txt

```diff
@@ -71,6 +71,7 @@ modes.
 :ino[remap] {lhs} {rhs}		|mapmode-i|	*:ino* *:inoremap*
 :ln[oremap] {lhs} {rhs}		|mapmode-l|	*:ln*  *:lnoremap*
 :cno[remap] {lhs} {rhs}		|mapmode-c|	*:cno* *:cnoremap*
+:tno[remap] {lhs} {rhs}   |mapmode-t| *:tno* *:tnoremap*
```

Comment:
There is no *mapmode-t*, so it needs to be added (see lines 291). It should be mentioned also at the end of the list in line 318.